data/json/itemgroups/stashes.json

```diff
@@ -82,6 +82,29 @@
       { "item": "lsd", "prob": 6, "count": [ 1, 5 ] }
     ]
   },
+  {
+    "type": "item_group",
+    "id": "stash_drugs_illegals",
+    "items": [
+      { "prob": 5, "group": "melatonin_tablet_bottle_plastic_pill_supplement_1_30" },
```

Comment:
Why is like the most normal sleep medication in the universe in a group with illegal drugs